src/neo/SmartContract/Native/NativeContract.cs

```diff
@@ -136,6 +136,7 @@ protected NativeContract()
             this.Nef = new NefFile
             {
                 Compiler = "neo-core-v3.0",
+                Source = string.Empty,
                 Tokens = Array.Empty<MethodToken>(),
```

Comment:
```suggestion
                Source = string.Empty,
                Version= string.Empty, // I think it would be great if we could have the contract version in the nef or somewhere
                Tokens = Array.Empty<MethodToken>(),
```

Reply:
It's not compatible with the old format. I think it's easy to add a `version()` method to contracts.